### PR TITLE
Fix 3D model positions in the 2D editor

### PR DESCRIPTION
**Pull request by D8H**

### CODE REVIEW

**Review by ClementPasteau — APPROVED** [review]
lgtm 👍

File: Extensions/3D/JsExtension.js [hunk 1]
```diff
@@ -2316,36 +2322,192 @@ module.exports = {
...
 
-        this._pixiObject.position.x = this._instance.getX() + width / 2;
-        this._pixiObject.position.y = this._instance.getY() + height / 2;
+        const originPoint = this.getOriginPoint();
+        this._pixiObject.position.x =
+          this._instance.getX() - width * (originPoint[0] - centerPoint[0]);
+        this._pixiObject.position.y =
+          this._instance.getY() - height * (originPoint[1] - centerPoint[1]);
         this._pixiObject.angle = this._instance.getAngle();
       }
```
Comment (ClementPasteau) [inline Extensions/3D/JsExtension.js:2513]: should we also add a getDefaultDepth() function? (mirrorring what's done for other models)

Comment (ClementPasteau) [reply Extensions/3D/JsExtension.js:2513]: I'm wondering if this is useful when drag an object on the scene and it creates the instance, to ensure the depth is properly set for the instance. (I fixed this issue for the 3D Cube in 2D)

Comment (D8H) [reply Extensions/3D/JsExtension.js:2513]: Good catch

**Pull request merged**